ros_buildfarm/argument.py

```diff
@@ -63,6 +63,14 @@ def add_argument_build_name(parser, build_file_type):
              build_file_type)
 
 
+def add_argument_env_vars(parser):
+    parser.add_argument(
+        '--env-vars',
+        nargs='*',
+        help="Environment variables as 'key=value' for Dockerfile "
+        'ENV directives')
```

Comment:
This line will need to be indented so that the single quote matches the column of the double quote in the line above.